### PR TITLE
Transition SwiftUI performance from CocoaPods to SwiftPM

### PR DESCRIPTION
**Pull request by jrcrespoh**

### CODE REVIEW

**Review by granluo — APPROVED** [review]

**Comment by paulb777** [comment]
Please fix the CI issue in the iOS tests:

`Missing secrets: tests did not run.` even though the tvOS tests do run.

File: .github/workflows/performance.yml [hunk 1]
```diff
@@ -53,63 +53,63 @@ jobs:
         run: |
           set -io pipefail
           EXIT_STATUS=0
-          source scripts/check_secrets.sh
-          cd performance/SwiftUIPerformanceQuickstart/PerformanceExample
-          bundle exec pod install --repo-update
+          ./scripts/check_secrets.sh
```
Comment (paulb777) [inline .github/workflows/performance.yml]: Might need to be `source scripts/check_secrets.sh`

Comment (jrcrespoh) [reply .github/workflows/performance.yml]: Yep, just reverted that change 👍

**Review by paulb777 — APPROVED** [review]
🚢

**Pull request merged**